send_knack_messages_to_esb.py

```diff
@@ -115,23 +115,8 @@ def build_template_dict(*, record, fields, activity_codes, outcome_codes):
     activity_name = template_dict["activity_name"]
     outcome_code = outcome_codes.get(activity_name, outcome_codes["default"])
     template_dict["csr_outcome_code"] = outcome_code
-
-    """311 and Knack to do not agree on what counts as a duplicate issue. This is because
```

Comment:
the removal of this comment and extra handling feels like a nice win. Is this a side effect of removing the redundant field or were duplicates part of the issue that y'all worked on today?

Reply:
Thanks for checking on this. This was a bonus side effect of removing the redundant field 😁